cassandra-diagnostics-reporter-kafka/src/main/java/io/smartcat/cassandra/diagnostics/reporter/KafkaReporter.java

```diff
@@ -49,7 +49,7 @@ public KafkaReporter(ReporterConfiguration configuration) {
         properties.put(ProducerConfig.VALUE_SERIALIZER_CLASS_CONFIG, StringSerializer.class.getName());
 
         producer = new KafkaProducer<>(properties);
-        partitionKey = Utils.getHostname();
+        partitionKey = Utils.getSystemname() + "_" + Utils.getHostname();
```

Comment:
Configuration is setting things in utils. We should not let measurement handle host and system name. It is global and our global source of truth is in Utils which is in commons.